src/core/Form/TextInput/TextInput.tsx

```diff
@@ -68,10 +71,14 @@ export interface TextInputProps
   type?: 'text' | 'email' | 'number' | 'password' | 'tel' | 'url';
   /** Input name */
   name?: string;
+  /** Controlled value */
+  value?: TextInputValue;
   /** Set components width to 100% */
   fullWidth?: boolean;
   /** Text to mark a field optional. Will be wrapped in parentheses and shown after labelText. */
   optionalText?: string;
+  /** Debounce time in milliseconds for onChange function. No debounce is applied if no value is given. */
+  debounce?: number;
   icon?: BaseIconKeys;
```

Comment:
Not related to this PR, but why is the icon here if it is not even documented?

Reply:
Probably just an oversight from earlier. I'll add it in this PR since it is such a small thing.